src/App.jsx

```diff
@@ -1,14 +1,23 @@
 import React from 'react';
-import './App.css';
+import { Sum } from './components/Sum/Sum';
+import './App.scss';
 
 export const App = () => (
   <>
-    <p>Sum of 2 and 3 is 5</p>
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove commented lines :) */}
+    <Sum
+      a={2}
+      b={3}
+    />
```

Comment:
```suggestion
    <Sum a={2} b={3}  />
```
it will be easier to read
write each prop on the new line if there are more than 3 props (the same as with HTML attributes)